solutions/community-deploy/CORTX-Deployment.md

```diff
@@ -58,19 +58,21 @@ You can use following command for RedHat family OS to install the Git.
 # git clone https://github.com/Seagate/cortx-re && cd $PWD/cortx-re/solutions/kubernetes
 ```
 
--  Create the host file in below format.
+-  Create the hosts file in the current directory.
 ```
-hostname=<hostname>,user=root,pass=<root-password>
+echo "hostname=<hostname>,user=root,pass=rootuserpassword" > hosts && cat hosts
 ```
+
 -  Script supports N-node deployment. Add entries for all the nodes with same format in hosts file. Node from first entry will be configured as Primary node. Example `hosts` file for multi-node setup is as below,
 ```
+vi hosts
```

Comment:
`vi` command not needed

Reply:
@subhalaxmisa, but they did not mentioned add `vi` command. anyway, it is not working properly if I copy it and paste it in my shell. could you please verify

Reply:
It's working for me. Maybe with the system changes it might not work. So on a safer side I am removing it.

![image](https://user-images.githubusercontent.com/104340707/173495378-6e9a0f67-d836-44a4-ba7e-80f59a8bf1b6.png)